sedumi.m

```diff
@@ -794,7 +794,7 @@
         info.err(4)=0;
     end
     %     Dual infeasibility
-    %info.err(3)=0.0; %s is not maintained explicitely
+    info.err(3)=0.0; %s is not maintained explicitely

```

Comment:
Basically, this change is not necessary, because `info.err=zeros(1,6);` above.  But you are right, it looks nicer without comment.